src/app/app.module.ts

```diff
@@ -20,6 +21,7 @@ export function HttpLoaderFactory(http: HttpClient) {
   ],
   imports: [
     BrowserModule,
+    MyScsModule,
```

Comment:
no need to import it here, it should work without it.
+ it breakes routing, if you open "localhost:4200", you are redirected thn to your new component instead of dashboard.